openfisca_aotearoa/tests/social_security/disability_allowance/income_limits.yaml

```diff
@@ -0,0 +1,213 @@
+- name: Disability allowance - single person without dependant children, age b/w 16-17 years (Clause 10)
+  period: 2022-W07
+  absolute_error_margin: 0
+  input:
+    persons:
+      Lily:
+        date_of_birth: 2006-01-01
+        social_security__income: 
+          2022-W07: 200
+          2022-W08: 300
+          2022-W09: 589
+    families:
+      Whanau:
+        principal: Lily
+  output:
+    age:
+      2022-02-20: 16
+    social_security__in_a_relationship:
```

Comment:
I'd tend to avoid these and test them where appropriate. If for some reason you want to be sure of the value of these, pass them as input.